README.md

```diff
@@ -240,14 +265,15 @@ ARGUMENTS
   COLLECTION  Collection name
 
 OPTIONS
+  --as=as                  Impersonate a user
   --batch-size=batch-size  [default: 2000] Maximum batch size (see limits.documentsFetchCount config)
   --editor                 Open an editor (EDITOR env variable) to edit the query before sending
   --help                   show CLI help
   --host=host              [default: localhost] Kuzzle server host
   --password=password      Kuzzle user password
   --path=path              Dump root directory
   --port=port              [default: 7512] Kuzzle server port
-  --protocol=protocol      [default: websocket] Kuzzle protocol (http or websocket)
+  --protocol=protocol      [default: ws] Kuzzle protocol (http or websocket)
```

Comment:
The list of available values need to be updated (for all commands) : `http or websocket` => `http or ws`